sky/cli.py

```diff
@@ -2995,26 +2995,45 @@ def spot_queue(all: bool, refresh: bool, skip_finished: bool):
       watch -n60 sky spot queue
     """
     click.secho('Fetching managed spot job statuses...', fg='yellow')
+    no_jobs_found_str = '  No jobs found.'
     try:
         job_table = core.spot_queue(refresh=refresh,
                                     skip_finished=skip_finished)
     except exceptions.ClusterNotUpError:
+        # TODO(mehul): handle skip_finished for the cached case. E.g., change
+        # {load,dump}_job_table_cache() to use structured data, and let
+        # format_job_table() to take skip_finished.
         cache = spot_lib.load_job_table_cache()
         if cache is not None:
             readable_time = log_utils.readable_time_duration(cache[0])
+            if not cache[1].strip():
+                table_str = no_jobs_found_str
+                cached_hint = ''
+            else:
+                table_str = cache[1]
+                # Show a helpful hint at the end, if cached table is non-empty.
+                cached_hint = (
```

Comment:
This feels repetitive given that the title of the table has the fact that it was cached and when it was last updated already. I think what might be more intuitive is something like this:

```
Fetching managed spot job statuses...
Job Status Table [last updated: now/x minutes ago]
{INSERT TABLE}
WARNING: spot-controller is stopped so the status table is not up to date
Use --refresh to view the latest table
```

Reply:
Reworded a bit: 

<img width="723" alt="Screen Shot 2023-01-14 at 20 25 46" src="https://user-images.githubusercontent.com/592670/212522880-c3e21d81-7db7-4745-b34f-254aeba07e1e.png">

We did intend it to feel repetitive due to user feedback in #1486:
> Maybe make it appear to the bottom of the content rather than to the top (or both) would help..

It should be ok especially if users have >1 page of jobs in this table. Thanks for pointing this out!